src/Module.Server/Modes/Dtv/CrpgDtvServer.cs

```diff
@@ -117,6 +117,25 @@ public override void OnMissionTick(float dt)
         }
     }
 
+    public override void OnScoreHit(
+    Agent affectedAgent,
+    Agent affectorAgent,
+    WeaponComponentData attackerWeapon,
+    bool isBlocked,
+    bool isSiegeEngineHit,
+    in Blow blow,
+    in AttackCollisionData collisionData,
+    float damagedHp,
+    float hitDistance,
+    float shotDifficulty)
+    {
+        if (affectedAgent.IsAIControlled && affectedAgent.Name == "The Viscount") // Viscount under attack
```

Comment:
Testing the name sounds fragile. You could check if it's a defender ai instead. This logic is already used elsewhere in the class.

Reply:
changed to `if (affectedAgent.IsAIControlled && affectedAgent.Team == Mission.DefenderTeam)`